spec/cancancan/model_adapters/neo4j_adapter_spec.rb

```diff
@@ -58,7 +56,7 @@
       article2 = Article.create!(published: true, secret: true)
       article3 = Article.create!(published: false, secret: true)
       Article.create!(published: false, secret: false)
-      expect(Article.accessible_by(@ability).to_a).to eq([article1, article2, article3])
+      expect(Article.accessible_by(@ability)).to contain_exactly(article1, article2, article3)
```

Comment:
without `order by` clause order is not guaranteed. Order could be achieved by passing a scope to the rule. Is that possible?